### PR TITLE
Add who region col

### PR DESCRIPTION
**Pull request by atatmaja**
## Briefly describe the feature or bug that this PR addresses.

Add who region col and refactor some code to use pandas vectorization

## Please link the Airtable ticket associated with this PR.

https://airtable.com/tbli2lWQHAqBa6ZcI/viwWvxb27XGuziw4c/rechqjK7EuisnUcT5?blocks=bipIkdtYHmboUfJut

## Describe the steps you took to test the feature/bugfix introduced by this PR.

Ran ETL locally, smoke tested app running locally

## Does any infrastructure work need to be done before this PR can be pushed to production?

Migration

### CODE REVIEW

File: app/database_etl/postgres_tables_handler/table_formatter.py [hunk 1]
```diff
@@ -17,33 +17,33 @@ def replace_null_string(x):
     return x
 
 
+def get_WHO_mapped_variables(country: str, gbd_mapping_df: pd.DataFrame):
+    # Get row in GBD mapping DF that corresponds to the input country
+    gbd_row = gbd_mapping_df[gbd_mapping_df['Country'] == country]
+    if not gbd_row.empty:
+        return gbd_row.iloc[0]['GBD Region'], gbd_row.iloc[0]['GBD Subregion'], gbd_row.iloc[0]['WHO region']
```
Comment (simonarocco) [inline app/database_etl/postgres_tables_handler/table_formatter.py:24]: good refactor. cleaner than before

**Review by simonarocco — APPROVED** [review]
good!

File: app/database_etl/postgres_tables_handler/table_formatter.py (hunk 1 above)
Comment (rahularoradfs) [reply app/database_etl/postgres_tables_handler/table_formatter.py:24]: blesssed refactor

File: app/database_etl/postgres_tables_handler/table_formatter.py [hunk 2]
```diff
@@ -17,33 +17,33 @@ def replace_null_string(x):
...
-            gbd_subregion_col.append(None)
+    df['gbd_region'], df['gbd_subregion'], df['who_region'] = \
+        zip(*df['country'].map(lambda x: get_WHO_mapped_variables(x, gbd_mapping_country)))
 
-            # Send slack message and log warning
-            body = f'No GBD region or subregion found for {country}'
+    # Add logging for countries without available GBD mappings
+    # Get set of all countries that exist in df but not in gbd_mapping_country
+    # See https://realpython.com/python-sets/#operators-vs-methods
+    countries_with_no_mapping = set(df['country']) - set(gbd_mapping_country['Country'])
```
Comment (rahularoradfs) [inline app/database_etl/postgres_tables_handler/table_formatter.py:39]: Does this also cover WHO mappings? GBD subregions?

Comment (rahularoradfs) [reply app/database_etl/postgres_tables_handler/table_formatter.py:39]: We should parse this so it drops none; see recent outputs in #dev-logging-etl

**Review by rahularoradfs — APPROVED** [review]
good stuff! minor comment re logging

File: app/database_etl/postgres_tables_handler/table_formatter.py (hunk 2 above)
Comment (atatmaja) [reply app/database_etl/postgres_tables_handler/table_formatter.py:39]: this covers all mappings in the csv (gbd_region, gbd_subregion, who_region)

Comment (atatmaja) [reply app/database_etl/postgres_tables_handler/table_formatter.py:39]: addressed review comments, merging after ETL runs successfully locally

**Pull request merged**